templates/zerver/help/change-your-org-name.md

```diff
@@ -0,0 +1,18 @@
+# Change your organization's name
+Only an administrator can change the name of the organization.
+
+Follow the following steps to change the name of the organization using an administrator account:
+* Click on the cog (![cog](/static/images/help/cog.png)) icon in the upper right corner
+of the right sidebar.
+
+* Choose **Administration** in the panel below:
+
+![admin-panel](/static/images/help/admin.png)
+
+* The following window will appear:
+![admin-window](/static/images/help/admin-window.png)
+
+* Edit your organization's name in the **Your organization's name** textbox.
```

Comment:
Are there any more details about this field? Is there a character limit? Are there any characters that can't be used in the organization name?